tools/travis.ps1

```diff
@@ -203,16 +220,6 @@ else
             }            
         }
 
-        # Determine whether the build passed
-        try {
-            # this throws if there was an error
-            Test-PSPesterResults -ResultObject $pesterPassThruObject
-            $result = "PASS"
-        }
-        catch {
-            $resultError = $_
-            $result = "FAIL"
-        }
```

Comment:
Isn't this change already done in #4807?

Reply:
I rebased and resolved the issue.